features/users/create-user-table.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * @desc   If table exists drop table then create it, if it does not exist just create it
+ * @param  {Object} dbConfig Shared database configuration object
+ * @return {Promise}      Promise with resolving or reject
+ */
+function createUserTable(dbConfig) {
+  return dbConfig.db
```

Comment:
If you always delete the table if it exists, how does this app keep it's data between executions?